src/types/emr/specimenDefinition/specimenDefinition.ts

```diff
@@ -85,8 +84,8 @@ export interface SpecimenDefinition {
   slug: string;
   derived_from_uri: string | null;
```

Comment:
```suggestion
  derived_from_uri?: string | null;
```

<img width="779" alt="image" src="https://github.com/user-attachments/assets/8a9163a2-2d3c-4234-bcfa-5423125718d0" />